index.js

```diff
@@ -0,0 +1,7 @@
+const app = require('./src/app');
+
+const APP_PORT = 3000;
+
+app.listen(APP_PORT, () => {
+  console.log(`Now serving your Express app at http://localhost:${APP_PORT}`); // eslint-disable-line
```

Comment:
Why have you disabled the lint here? What error message were you getting from the linter?

It's easy when you are learning to start disabling things if you want your red squigglies to disappear, but it will help you and get you into good practices if you learn to read what the lint error is telling you and learn how to resolve it without disabling.